FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/FreeRTOS_IP_Private.h

```diff
@@ -721,9 +721,10 @@ int32_t lTCPAddRxdata(FreeRTOS_Socket_t *pxSocket, size_t uxOffset, const uint8_
 void vSocketWakeUpUser( FreeRTOS_Socket_t *pxSocket );
 
 /*
- * Some helping function, their meaning should be clear
+ * Some helping function, their meaning should be clear.
+ * Utility functions can be used anywhere in the source. Cannot be defined
```

Comment:
This comment isn't very clear and contains a sentence fragment (_"Cannot be defined conditionally"_).  Can we rephrase this to be more clear?

Reply:
Re-worded. Thanks.